radio/lib/radio.ts

```diff
@@ -34,6 +35,8 @@ export class Radio extends LitElement {
   /** @nocollapse */
   static formAssociated = true;
 
+  private maskId = `cutout${++maskId}`;
```

Comment:
Can you add a short comment explaining that this is due to a Safari bug? Once it's fixed in webkit, we can remove it.